<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="7BB9LQ2" name="com.ca.tds.main.TDSMethodURL_TC" tests="66" failures="30" timestamp="10 Jan 2020 13:27:45 GMT" time="177.560" errors="0">
  <testcase name="testArequestAPI" time="1.283" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API response schema validation failed.&lt;br&gt;required key [errorMessageType] not found&lt;br&gt; api response : {&quot;errorComponent&quot;:&quot;S&quot;,&quot;errorDescription&quot;:&quot;Invalid messageType field or messageType field missing&quot;,&quot;messageType&quot;:&quot;Erro&quot;,&quot;errorCode&quot;:&quot;101&quot;,&quot;errorDetail&quot;:&quot;Invalid messageType field or messageType field missing&quot;,&quot;messageVersion&quot;:&quot;2.1.0&quot;,&quot;callerTxnRefID&quot;:&quot;341b7153-da86-4543-860f-2ba47f0168a5&quot;,&quot;threeDSServerTransID&quot;:&quot;ce860a3b-f4e2-416a-82cd-6a2e0e1d7f96&quot;}">
      <![CDATA[java.lang.AssertionError: Three DS method URL API response schema validation failed.<br>required key [errorMessageType] not found<br> api response : {"errorComponent":"S","errorDescription":"Invalid messageType field or messageType field missing","messageType":"Erro","errorCode":"101","errorDetail":"Invalid messageType field or messageType field missing","messageVersion":"2.1.0","callerTxnRefID":"341b7153-da86-4543-860f-2ba47f0168a5","threeDSServerTransID":"ce860a3b-f4e2-416a-82cd-6a2e0e1d7f96"}
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:84)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="2.899" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.828" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.899" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="1.380" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API response schema validation failed.&lt;br&gt;required key [errorMessageType] not found&lt;br&gt; api response : {&quot;errorComponent&quot;:&quot;S&quot;,&quot;errorDescription&quot;:&quot;Invalid messageType field or messageType field missing&quot;,&quot;messageType&quot;:&quot;Erro&quot;,&quot;errorCode&quot;:&quot;101&quot;,&quot;errorDetail&quot;:&quot;Invalid messageType field or messageType field missing&quot;,&quot;messageVersion&quot;:&quot;2.1.0&quot;,&quot;callerTxnRefID&quot;:&quot;341b7153-da86-4543-860f-2ba47f0168a5&quot;,&quot;threeDSServerTransID&quot;:&quot;0a45ddf5-53cc-4da7-b428-9493f171f439&quot;}">
      <![CDATA[java.lang.AssertionError: Three DS method URL API response schema validation failed.<br>required key [errorMessageType] not found<br> api response : {"errorComponent":"S","errorDescription":"Invalid messageType field or messageType field missing","messageType":"Erro","errorCode":"101","errorDetail":"Invalid messageType field or messageType field missing","messageVersion":"2.1.0","callerTxnRefID":"341b7153-da86-4543-860f-2ba47f0168a5","threeDSServerTransID":"0a45ddf5-53cc-4da7-b428-9493f171f439"}
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:84)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="3.008" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.897" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table">
      <![CDATA[java.lang.AssertionError: Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.BaseClassTDS.assertTDSMethodRes(BaseClassTDS.java:364)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:81)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="2.805" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table">
      <![CDATA[java.lang.AssertionError: Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.BaseClassTDS.assertTDSMethodRes(BaseClassTDS.java:364)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:81)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="2.965" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.865" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="3.013" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.839" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.826" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [201] but found [303],
	expected [caMerchantID] but found [Access Denied, Invalid End Point]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [201] but found [303],
expected [caMerchantID] but found [Access Denied, Invalid End Point]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:82)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="2.806" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="1.711" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API response schema validation failed.&lt;br&gt;required key [errorMessageType] not found&lt;br&gt; api response : {&quot;errorComponent&quot;:&quot;S&quot;,&quot;errorDescription&quot;:&quot;Invalid messageType field or messageType field missing&quot;,&quot;messageType&quot;:&quot;Erro&quot;,&quot;errorCode&quot;:&quot;101&quot;,&quot;errorDetail&quot;:&quot;Invalid messageType field or messageType field missing&quot;,&quot;messageVersion&quot;:&quot;2.1.0&quot;,&quot;callerTxnRefID&quot;:&quot;341b7153-da86-4543-860f-2ba47f0168a5&quot;,&quot;threeDSServerTransID&quot;:&quot;942a9011-6cc7-429f-857d-372c27eb9b5d&quot;}">
      <![CDATA[java.lang.AssertionError: Three DS method URL API response schema validation failed.<br>required key [errorMessageType] not found<br> api response : {"errorComponent":"S","errorDescription":"Invalid messageType field or messageType field missing","messageType":"Erro","errorCode":"101","errorDetail":"Invalid messageType field or messageType field missing","messageVersion":"2.1.0","callerTxnRefID":"341b7153-da86-4543-860f-2ba47f0168a5","threeDSServerTransID":"942a9011-6cc7-429f-857d-372c27eb9b5d"}
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:84)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="2.898" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.898" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.953" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [203] but found [303],
	expected [caMerchantID] but found [Access Denied, Invalid End Point]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [203] but found [303],
expected [caMerchantID] but found [Access Denied, Invalid End Point]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:82)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="2.812" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table">
      <![CDATA[java.lang.AssertionError: Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.BaseClassTDS.assertTDSMethodRes(BaseClassTDS.java:364)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:81)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="1.305" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API response schema validation failed.&lt;br&gt;required key [errorMessageType] not found&lt;br&gt; api response : {&quot;errorComponent&quot;:&quot;S&quot;,&quot;errorDescription&quot;:&quot;MessageType field missing&quot;,&quot;messageType&quot;:&quot;Erro&quot;,&quot;errorCode&quot;:&quot;201&quot;,&quot;errorDetail&quot;:&quot;MessageType field missing&quot;,&quot;messageVersion&quot;:&quot;2.1.0&quot;,&quot;callerTxnRefID&quot;:&quot;341b7153-da86-4543-860f-2ba47f0168a5&quot;,&quot;threeDSServerTransID&quot;:&quot;7291b24e-9541-47e6-bc4a-79f261789b94&quot;}">
      <![CDATA[java.lang.AssertionError: Three DS method URL API response schema validation failed.<br>required key [errorMessageType] not found<br> api response : {"errorComponent":"S","errorDescription":"MessageType field missing","messageType":"Erro","errorCode":"201","errorDetail":"MessageType field missing","messageVersion":"2.1.0","callerTxnRefID":"341b7153-da86-4543-860f-2ba47f0168a5","threeDSServerTransID":"7291b24e-9541-47e6-bc4a-79f261789b94"}
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:84)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="3.257" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.877" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.875" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.859" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table">
      <![CDATA[java.lang.AssertionError: Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.BaseClassTDS.assertTDSMethodRes(BaseClassTDS.java:364)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:81)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="2.888" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.882" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="3.804" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table">
      <![CDATA[java.lang.AssertionError: Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.BaseClassTDS.assertTDSMethodRes(BaseClassTDS.java:364)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:81)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="2.915" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.833" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.085" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API response schema validation failed.&lt;br&gt;required key [errorMessageType] not found&lt;br&gt; api response : {&quot;errorComponent&quot;:&quot;S&quot;,&quot;errorDescription&quot;:&quot;Invalid messageType field or messageType field missing&quot;,&quot;messageType&quot;:&quot;Erro&quot;,&quot;errorCode&quot;:&quot;101&quot;,&quot;errorDetail&quot;:&quot;Invalid messageType field or messageType field missing&quot;,&quot;messageVersion&quot;:&quot;2.1.0&quot;,&quot;callerTxnRefID&quot;:&quot;341b7153-da86-4543-860f-2ba47f0168a5&quot;,&quot;threeDSServerTransID&quot;:&quot;fecac5d8-e7f6-4962-b8b7-9167a4c7a193&quot;}">
      <![CDATA[java.lang.AssertionError: Three DS method URL API response schema validation failed.<br>required key [errorMessageType] not found<br> api response : {"errorComponent":"S","errorDescription":"Invalid messageType field or messageType field missing","messageType":"Erro","errorCode":"101","errorDetail":"Invalid messageType field or messageType field missing","messageVersion":"2.1.0","callerTxnRefID":"341b7153-da86-4543-860f-2ba47f0168a5","threeDSServerTransID":"fecac5d8-e7f6-4962-b8b7-9167a4c7a193"}
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:84)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="2.899" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.912" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table">
      <![CDATA[java.lang.AssertionError: Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.BaseClassTDS.assertTDSMethodRes(BaseClassTDS.java:364)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:81)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="2.947" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [203] but found [303],
	expected [caMerchantID] but found [Access Denied, Invalid End Point]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [203] but found [303],
expected [caMerchantID] but found [Access Denied, Invalid End Point]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:82)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="2.908" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="1.241" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API response schema validation failed.&lt;br&gt;required key [errorMessageType] not found&lt;br&gt; api response : {&quot;errorComponent&quot;:&quot;S&quot;,&quot;errorDescription&quot;:&quot;Invalid messageType field or messageType field missing&quot;,&quot;messageType&quot;:&quot;Erro&quot;,&quot;errorCode&quot;:&quot;101&quot;,&quot;errorDetail&quot;:&quot;Invalid messageType field or messageType field missing&quot;,&quot;messageVersion&quot;:&quot;2.1.0&quot;,&quot;callerTxnRefID&quot;:&quot;341b7153-da86-4543-860f-2ba47f0168a5&quot;,&quot;threeDSServerTransID&quot;:&quot;5c9edeae-5907-4ecb-a039-146a100f0f9a&quot;}">
      <![CDATA[java.lang.AssertionError: Three DS method URL API response schema validation failed.<br>required key [errorMessageType] not found<br> api response : {"errorComponent":"S","errorDescription":"Invalid messageType field or messageType field missing","messageType":"Erro","errorCode":"101","errorDetail":"Invalid messageType field or messageType field missing","messageVersion":"2.1.0","callerTxnRefID":"341b7153-da86-4543-860f-2ba47f0168a5","threeDSServerTransID":"5c9edeae-5907-4ecb-a039-146a100f0f9a"}
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:84)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="1.388" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API response schema validation failed.&lt;br&gt;required key [errorMessageType] not found&lt;br&gt; api response : {&quot;errorComponent&quot;:&quot;S&quot;,&quot;errorDescription&quot;:&quot;MessageType field missing&quot;,&quot;messageType&quot;:&quot;Erro&quot;,&quot;errorCode&quot;:&quot;201&quot;,&quot;errorDetail&quot;:&quot;MessageType field missing&quot;,&quot;messageVersion&quot;:&quot;2.1.0&quot;,&quot;callerTxnRefID&quot;:&quot;341b7153-da86-4543-860f-2ba47f0168a5&quot;,&quot;threeDSServerTransID&quot;:&quot;a9a4c684-e806-42c4-b1a0-0e8cb4a3436d&quot;}">
      <![CDATA[java.lang.AssertionError: Three DS method URL API response schema validation failed.<br>required key [errorMessageType] not found<br> api response : {"errorComponent":"S","errorDescription":"MessageType field missing","messageType":"Erro","errorCode":"201","errorDetail":"MessageType field missing","messageVersion":"2.1.0","callerTxnRefID":"341b7153-da86-4543-860f-2ba47f0168a5","threeDSServerTransID":"a9a4c684-e806-42c4-b1a0-0e8cb4a3436d"}
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:84)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="2.855" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="3.195" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table">
      <![CDATA[java.lang.AssertionError: Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.BaseClassTDS.assertTDSMethodRes(BaseClassTDS.java:364)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:81)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="2.889" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="3.309" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table">
      <![CDATA[java.lang.AssertionError: Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.BaseClassTDS.assertTDSMethodRes(BaseClassTDS.java:364)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:81)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="3.108" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table">
      <![CDATA[java.lang.AssertionError: Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.BaseClassTDS.assertTDSMethodRes(BaseClassTDS.java:364)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:81)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="3.047" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [201] but found [303],
	expected [caMerchantID] but found [Access Denied, Invalid End Point]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [201] but found [303],
expected [caMerchantID] but found [Access Denied, Invalid End Point]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:82)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="2.903" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.896" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="1.698" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API response schema validation failed.&lt;br&gt;required key [errorMessageType] not found&lt;br&gt; api response : {&quot;errorComponent&quot;:&quot;S&quot;,&quot;errorDescription&quot;:&quot;Invalid messageType field or messageType field missing&quot;,&quot;messageType&quot;:&quot;Erro&quot;,&quot;errorCode&quot;:&quot;101&quot;,&quot;errorDetail&quot;:&quot;Invalid messageType field or messageType field missing&quot;,&quot;messageVersion&quot;:&quot;2.1.0&quot;,&quot;callerTxnRefID&quot;:&quot;341b7153-da86-4543-860f-2ba47f0168a5&quot;,&quot;threeDSServerTransID&quot;:&quot;30929e6b-1061-44d8-8b1e-3f703a28b1dd&quot;}">
      <![CDATA[java.lang.AssertionError: Three DS method URL API response schema validation failed.<br>required key [errorMessageType] not found<br> api response : {"errorComponent":"S","errorDescription":"Invalid messageType field or messageType field missing","messageType":"Erro","errorCode":"101","errorDetail":"Invalid messageType field or messageType field missing","messageVersion":"2.1.0","callerTxnRefID":"341b7153-da86-4543-860f-2ba47f0168a5","threeDSServerTransID":"30929e6b-1061-44d8-8b1e-3f703a28b1dd"}
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:84)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="1.312" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API response schema validation failed.&lt;br&gt;required key [errorMessageType] not found&lt;br&gt; api response : {&quot;errorComponent&quot;:&quot;S&quot;,&quot;errorDescription&quot;:&quot;MessageType field missing&quot;,&quot;messageType&quot;:&quot;Erro&quot;,&quot;errorCode&quot;:&quot;201&quot;,&quot;errorDetail&quot;:&quot;MessageType field missing&quot;,&quot;messageVersion&quot;:&quot;2.1.0&quot;,&quot;callerTxnRefID&quot;:&quot;341b7153-da86-4543-860f-2ba47f0168a5&quot;,&quot;threeDSServerTransID&quot;:&quot;ed4e91a7-d66e-4793-a00e-36e33dd50f89&quot;}">
      <![CDATA[java.lang.AssertionError: Three DS method URL API response schema validation failed.<br>required key [errorMessageType] not found<br> api response : {"errorComponent":"S","errorDescription":"MessageType field missing","messageType":"Erro","errorCode":"201","errorDetail":"MessageType field missing","messageVersion":"2.1.0","callerTxnRefID":"341b7153-da86-4543-860f-2ba47f0168a5","threeDSServerTransID":"ed4e91a7-d66e-4793-a00e-36e33dd50f89"}
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:84)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="2.846" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="1.213" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API response schema validation failed.&lt;br&gt;required key [errorMessageType] not found&lt;br&gt; api response : {&quot;errorComponent&quot;:&quot;S&quot;,&quot;errorDescription&quot;:&quot;Invalid messageType field or messageType field missing&quot;,&quot;messageType&quot;:&quot;Erro&quot;,&quot;errorCode&quot;:&quot;101&quot;,&quot;errorDetail&quot;:&quot;Invalid messageType field or messageType field missing&quot;,&quot;messageVersion&quot;:&quot;2.1.0&quot;,&quot;callerTxnRefID&quot;:&quot;341b7153-da86-4543-860f-2ba47f0168a5&quot;,&quot;threeDSServerTransID&quot;:&quot;61c14cc7-6a22-4952-af5a-578de8b685ff&quot;}">
      <![CDATA[java.lang.AssertionError: Three DS method URL API response schema validation failed.<br>required key [errorMessageType] not found<br> api response : {"errorComponent":"S","errorDescription":"Invalid messageType field or messageType field missing","messageType":"Erro","errorCode":"101","errorDetail":"Invalid messageType field or messageType field missing","messageVersion":"2.1.0","callerTxnRefID":"341b7153-da86-4543-860f-2ba47f0168a5","threeDSServerTransID":"61c14cc7-6a22-4952-af5a-578de8b685ff"}
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:84)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="2.908" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [201] but found [303],
	expected [caMerchantID] but found [Access Denied, Invalid End Point]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [201] but found [303],
expected [caMerchantID] but found [Access Denied, Invalid End Point]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:82)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="1.346" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API response schema validation failed.&lt;br&gt;required key [errorMessageType] not found&lt;br&gt; api response : {&quot;errorComponent&quot;:&quot;S&quot;,&quot;errorDescription&quot;:&quot;Invalid messageType field or messageType field missing&quot;,&quot;messageType&quot;:&quot;Erro&quot;,&quot;errorCode&quot;:&quot;101&quot;,&quot;errorDetail&quot;:&quot;Invalid messageType field or messageType field missing&quot;,&quot;messageVersion&quot;:&quot;2.1.0&quot;,&quot;callerTxnRefID&quot;:&quot;341b7153-da86-4543-860f-2ba47f0168a5&quot;,&quot;threeDSServerTransID&quot;:&quot;9f9e196a-f9d3-4e67-bb48-bfe25300bb74&quot;}">
      <![CDATA[java.lang.AssertionError: Three DS method URL API response schema validation failed.<br>required key [errorMessageType] not found<br> api response : {"errorComponent":"S","errorDescription":"Invalid messageType field or messageType field missing","messageType":"Erro","errorCode":"101","errorDetail":"Invalid messageType field or messageType field missing","messageVersion":"2.1.0","callerTxnRefID":"341b7153-da86-4543-860f-2ba47f0168a5","threeDSServerTransID":"9f9e196a-f9d3-4e67-bb48-bfe25300bb74"}
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:84)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="2.918" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [203] but found [303],
	expected [caMerchantID] but found [Access Denied, Invalid End Point]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [203] but found [303],
expected [caMerchantID] but found [Access Denied, Invalid End Point]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:82)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="5.164" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.882" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.837" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.911" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table">
      <![CDATA[java.lang.AssertionError: Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.BaseClassTDS.assertTDSMethodRes(BaseClassTDS.java:364)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:81)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="2.913" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.863" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table">
      <![CDATA[java.lang.AssertionError: Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.BaseClassTDS.assertTDSMethodRes(BaseClassTDS.java:364)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:81)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="1.274" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API response schema validation failed.&lt;br&gt;required key [errorMessageType] not found&lt;br&gt; api response : {&quot;errorComponent&quot;:&quot;S&quot;,&quot;errorDescription&quot;:&quot;Invalid messageType field or messageType field missing&quot;,&quot;messageType&quot;:&quot;Erro&quot;,&quot;errorCode&quot;:&quot;101&quot;,&quot;errorDetail&quot;:&quot;Invalid messageType field or messageType field missing&quot;,&quot;messageVersion&quot;:&quot;2.1.0&quot;,&quot;callerTxnRefID&quot;:&quot;341b7153-da86-4543-860f-2ba47f0168a5&quot;,&quot;threeDSServerTransID&quot;:&quot;0b831f5b-b52b-4a68-8b87-aea2cffcabc3&quot;}">
      <![CDATA[java.lang.AssertionError: Three DS method URL API response schema validation failed.<br>required key [errorMessageType] not found<br> api response : {"errorComponent":"S","errorDescription":"Invalid messageType field or messageType field missing","messageType":"Erro","errorCode":"101","errorDetail":"Invalid messageType field or messageType field missing","messageVersion":"2.1.0","callerTxnRefID":"341b7153-da86-4543-860f-2ba47f0168a5","threeDSServerTransID":"0b831f5b-b52b-4a68-8b87-aea2cffcabc3"}
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:84)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
  <testcase name="testArequestAPI" time="2.848" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.870" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.889" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.871" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.788" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.849" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.891" classname="com.ca.tds.main.TDSMethodURL_TC"/>
  <testcase name="testArequestAPI" time="2.807" classname="com.ca.tds.main.TDSMethodURL_TC">
    <failure type="java.lang.AssertionError" message="Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table">
      <![CDATA[java.lang.AssertionError: Three DS method URL API threeDSServerTransID not found in MTDERRORMSG table
at org.testng.Assert.fail(Assert.java:96)
at com.ca.tds.main.BaseClassTDS.assertTDSMethodRes(BaseClassTDS.java:364)
at com.ca.tds.main.TDSMethodURL_TC.testArequestAPI(TDSMethodURL_TC.java:81)
at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- testArequestAPI -->
</testsuite> <!-- com.ca.tds.main.TDSMethodURL_TC -->
